SpatialGDK/Source/Private/Interop/SpatialReceiver.cpp

```diff
@@ -126,6 +126,8 @@ void USpatialReceiver::OnAddComponent(Worker_AddComponentOp& Op)
 	case SpatialConstants::POSITION_COMPONENT_ID:
 	case SpatialConstants::PERSISTENCE_COMPONENT_ID:
 	case SpatialConstants::ROTATION_COMPONENT_ID:
+	case SpatialConstants::PLAYER_SPAWNER_COMPONENT_ID:
```

Comment:
I'm thinking this switch is getting pretty gross, it was already gross. Maybe clean up at some point